<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyWeChat</class>
 <widget class="QDialog" name="MyWeChat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>726</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyWeChat</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>45</width>
       <height>500</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>45</width>
       <height>500</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="chat_images.qrc">:/images/chat1.png</pixmap>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget1" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>250</horstretch>
       <verstretch>50</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>500</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>250</width>
       <height>500</height>
      </size>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>250</width>
        <height>65</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>65</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>65</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="chat_images.qrc">:/images/chat2.png</pixmap>
      </property>
     </widget>
     <widget class="QTableWidget" name="userListTableWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>70</y>
        <width>250</width>
        <height>441</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>441</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>441</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="showGrid">
       <bool>false</bool>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>250</number>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
     <widget class="QPushButton" name="searchPushButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>22</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>刷新在线列表</string>
      </property>
     </widget>
     <zorder>label_2</zorder>
     <zorder>searchPushButton</zorder>
     <zorder>userListTableWidget</zorder>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget2" native="true">
     <property name="minimumSize">
      <size>
       <width>431</width>
       <height>500</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>431</width>
       <height>500</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>65</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="userLabel">
           <property name="minimumSize">
            <size>
             <width>121</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>121</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="chat_images.qrc">:/images/chat6_1.png</pixmap>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="chatTextBrowser"/>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="chat_images.qrc">:/images/chat9.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="chatTextEdit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>97</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>97</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sendPushButton">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>发送</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="chat_images.qrc"/>
 </resources>
 <connections/>
</ui>
